src/Util/Configuration.php

```diff
@@ -924,6 +948,19 @@ public function getPHPUnitConfiguration(): array
             }
         }
 
+        if ($root->hasAttribute('executionOrderDefects')) {
+            switch ((string) $root->getAttribute('executionOrderDefects')) {
+                case 'defectsFirst':
+                    $result['executionOrderDefects'] = TestSuiteSorter::ORDER_DEFECTS_FIRST;
+
+                    break;
+
+                default:
+                    $result['executionOrderDefects'] = TestSuiteSorter::ORDER_DEFAULT;
+
```

Comment:
dummy lines doesn't look needed here

Reply:
This supports when somebody uses the config attribute `executionOrderDefects='default'` which can happen when automated pipelines do something like `executionOrderDefects='$value'`. I have added a test for it.

Reply:
I meant the empty ("dummy") line number 960

Reply:
I thought that was way too simple! Even your comments on whitespace scare me into making more tests haha